packages/block-library/src/template-part/edit/label.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
**Note:**

We can definitely talk about refactoring this for reusability if we end up moving forward with template part labels